### ISSUE TITLE
Incorrect Event Data

### ISSUE THREAD

**Issue by JacksonMaxfield** [CouncilDataProject/cdptools#116]
Event Id: `d07e833e-67d8-4260-bcfa-3269c7f9f70c`

Labeled as: "Finance and Neighborhoods Committee"
Should be: "Gender Equity, Safe Communities, ..."

**Comment by aminamos** [comment]
@JacksonMaxfield I'm not sure how the pipeline works/populates the database, but the row with that event ID has details for multiple meetings. The legistar link in the event object is: https://seattle.legistar.com/MeetingDetail.aspx?LEGID=4117&GID=393&G=FFE3B678-CEF6-4197-84AC-5204EA4CFC0C

But the Seattle Channel link in the event object is: http://www.seattlechannel.org/mayor-and-council/city-council/2018/2019-gender-equity-safe-communities-new-americans-and-education-committee?videoid=x106736

The events happened on the same day (9/18/2019) but the Finance meeting was at 12:00pm and the Gender/Education meeting happened at 6:00pm.

**Comment by JacksonMaxfield** [comment]
Here is the relevant code of the pipeline that is doing it:

https://github.com/CouncilDataProject/cdptools/blob/master/cdptools/event_scrapers/seattle_event_scraper.py#L405

In short, we are doing an ordered set text diff between the minutes shown on SeattleChannel and the minutes found on legistar for any event that day and saying "which one has the least difference. This is the first time that I have found an event it has failed on. _But_, it may be because the script ran prior to the event being uploaded. So when it ran, it only saw the Finance Meeting and the finance meeting returned a text difference score of less than 50% so we approved it.

I would assume I can just delete the event in the database and rerun the processing and it will find the right event as they are now both available in Legistar.
